### ISSUE TITLE
Use with cordova // Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by davidanquetin** [drien/jquery-flipster#74]
Hello
i would to use Flipster in a Cordova project, 
in my index.html are included jquery, then flipster
```
<script type="text/javascript" src="cordova.js"></script>
<script type="text/javascript" src="js/vendor/jquery.min.js"></script>
<script type="text/javascript" src="js/vendor/jquery.flipster.min.js"></script>
```

I create flipster instance with 
`$('#coverflow-container').flipster({});`

If I only preview my page without cordova.js, flipster works.
But if i run my project with cordova, instanciation of flipster make an error : 

> Uncaught RangeError: Maximum call stack size exceeded

I know flipster is the problem, because when i remove the instanciation, the problem does not appear anymore.

I tried to make the instanciation before or after the "deviceReady" event of cordova, but it dose not change the problem... 

Have you an idea of this ?

Thanks
David

**Comment by shshaw** [comment]
I don't know that I've heard of Cordova before. [Is this the Cordova you're using?](https://cordova.apache.org/)

**Comment by davidanquetin** [comment]
@shshaw : yes it is !

**Comment by aniruddhbk** [comment]
I am facing the same issue not while using it in Cordova though.

Also reported here
http://stackoverflow.com/questions/34081441/uncaught-rangeerror-maximum-call-stack-size-exceeded-with-flipster

**Comment by shshaw** [comment]
Very strange. I'll look into this further. If you have any more information
(browsers affected or a live demo) that would be greatly appreciated.

**Comment by kulli2000** [comment]
Sorry, it seams it was my mistake, it's working now very fine!
Just I added background-color because without stays a small corner from images

**Comment by shshaw** [comment]
@kulli2000 Are you referring to issue #74 or issue #77?

**Comment by kulli2000** [comment]
@shshaw : #77 - thank you

**Comment by travelwifisas** [comment]
Same issue here but when loaded via a directive from angularJS.

**Comment by shshaw** [comment]
Merging with #79 which has more detail about the possible cause. This is all related, no doubt.

**Comment by shshaw** [comment]
Fixed in [1.1.1](https://github.com/drien/jquery-flipster/releases/tag/1.1.1). If no width is given, a `setInterval` is run every 500ms to trigger the `resize` event again instead of running in an infinite loop. 

If you want it to run right away and know when it will be visible, you can simply call the `jump` method with no parameter and resize will be triggered as usual.

Don't run this code with the last version of Flipster, it will crash your browser, but is perfectly fine in 1.1.1!

```js
$('body ').hide();

var flipContainer = $('.flipster').flipster();

setTimeout(function(){ 
   $('body').show();
   flipContainer.flipster('jump');
},1800);

```

**Comment by davidanquetin** [comment]
Thanks !
